tests/unit/test_experiments/test_experiment.py

```diff
@@ -38,19 +38,18 @@ def test_read_strings(self):
                 "Run US06 (V) for 5 minutes",
                 "Run US06 (W) for 0.5 hours",
             ],
-            {"test": "test"}, drive_cycles={"US06": drive_cycle},
+            {"test": "test"},
+            drive_cycles={"US06": drive_cycle},
```

Comment:
do we have an example anywhere of using a drive cycle in an experiment? or doesn't it actually work in a simulation yet? I'm not up to date with the progress there

Reply:
I think it works but need to double check